src/ActionsMenu.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import type {IntlShape} from 'react-intl';
+
+import ActionsMenuToggle from './ActionsMenuToggle';
+import ActionsMenuItem from './ActionsMenuItem';
+import FocusTrapManager from './FocusTrapManager';
+
+import './ActionsMenu.scss';
+
+import type {ActionToggleRegister} from './types';
+
+type ActionsMenuProps = {
+    intl: IntlShape,
+    changeHandler?: (event: Event, commandName: string) => void,
+    editingDisabled?: boolean,
+    // TODO: Flesh this definition out.
+    menuItems: {},
+    allowedActions: ActionToggleRegister,
+    usedActions: ActionToggleRegister
+};
+
+type ActionsMenuState = {
+    showMenu: boolean
+};
+
+class ActionsMenu extends React.Component<ActionsMenuProps, ActionsMenuState> {
+    focusTrapManager: FocusTrapManager;
+
+    static defaultProps = {
+        changeHandler: () => {},
+        editingDisabled: false,
+        usedActions: {},
+        menuItems: {
+            // TODO: Uncomment these out once C2LC-206 is merged.
+            // backward1: {
+            //     isAllowed: true,
+            //     labelKey: "Command.backward1"
+            // },
+            // backward2: {
+            //     isAllowed: true,
+            //     labelKey: "Command.backward2"
+            // },
+            // backward3: {
+            //     isAllowed: true,
+            //     labelKey: "Command.backward3"
+            // },
+            forward1: {
+                isAllowed: true,
+                labelKey: "Command.forward1"
+            },
+            forward2: {
+                isAllowed: true,
+                labelKey: "Command.forward2"
+            },
+            forward3: {
+                isAllowed: true,
+                labelKey: "Command.forward3"
+            },
+            left45: {
+                isAllowed: true,
+                labelKey: "Command.left45"
+            },
+            left90: {
+                isAllowed: true,
+                labelKey: "Command.left90"
+            },
+            left180: {
+                isAllowed: true,
+                labelKey: "Command.left180"
+            },
+            right45: {
+                isAllowed: true,
+                labelKey: "Command.right45"
+            },
+            right90: {
+                isAllowed: true,
+                labelKey: "Command.right90"
+            },
+            right180: {
+                isAllowed: true,
+                labelKey: "Command.right180"
+            }
+        }
+    }
+    constructor (props: ActionsMenuProps) {
+        super(props);
+        this.state = { showMenu: false };
+        this.focusTrapManager = new FocusTrapManager();
+        this.focusTrapManager.setFocusTrap(this.handleCloseActionMenuFocusTrap, [".ActionsMenuItem__button"], ".ActionsMenu__toggle-button");
```

Comment:
Now we are using checkboxes as focusable item, we need to update this with checkbox's className.
`
this.focusTrapManager.setFocusTrap(this.handleCloseActionMenuFocusTrap, [".ActionsMenuItem__checkbox"], ".ActionsMenu__toggle-button");
`

Reply:
Done.